code/jumpnode/jumpnode.h

```diff
@@ -36,6 +36,7 @@ class CJumpNode
 
     int	m_modelnum;
     int	m_objnum;						// objnum of this jump node
+	int m_polymodel_instance_num {-1}; // polymodel instance number, used for rotations 
```

Comment:
In order to keep things consistent, I recommend adding default initializers to the rest of the class fields too.  Then you can remove the initializers from the `CJumpNode()` and `CJumpNode(vec3d* position)` constructors.

Reply:
Good point.

Reply:
Changes made.  If you want them squashed instead of a separate commit that's fine.  I just figured it would help anyone tracing the history to have a separate commit here when figuring out why the changes were made.

Reply:
Good. :+1: No need to squash; I don't think squashing is required in most cases.